client/blocks/reader-post-card/style.scss

```diff
@@ -0,0 +1,288 @@
+// Custom breakpoints needed to match original design
+
+$reader-post-card-breakpoint-xlarge: "( min-width: 1040px )";
+$reader-post-card-breakpoint-large: "( min-width: 961px ) and ( max-width: 1040px )";
+$reader-post-card-breakpoint-medium: "( min-width: 661px ) and ( max-width: 940px )";
+$reader-post-card-breakpoint-small: "( max-width: 550px )";
+
+.reader-post-card.card {
+	border-bottom: 1px solid lighten( $gray, 30% );
+	box-shadow: none;
+	padding: 16px 0 0;
+
+	@include breakpoint( "<660px" ) {
+		margin: 16px;
+	}
+
+	&.has-thumbnail {
+
+		.reader-post-card__post-details {
+			flex: 1;
+			margin-top: -13px;
+			padding-left: 265px;
+
+			@media #{$reader-post-card-breakpoint-large} {
+				padding-left: 205px;
+			}
+
+			@include breakpoint( "660px-960px" ) {
+				padding-left: 0;
+			}
+
+			@include breakpoint( "<660px" ) {
+				padding-left: 205px;
+			}
+
+			@include breakpoint( "<480px" ) {
+				padding-left: 0;
+			}
```

Comment:
I'm not sure how far this applies, but our coding guidelines dictate that we should be _mobile-first_, in which case `padding-left: 0` should be the default and the `265px` modifier should be allocated to some `>` breakpoint